ScanbotBarcodeSDKExample/BarcodeSDK.MAUI.Example/BarcodeSDK.MAUI.Example.csproj

```diff
@@ -50,39 +49,14 @@
         <!-- Raw Assets (also remove the "Resources\Raw" prefix) -->
         <MauiAsset Include="Resources\Raw\**" LogicalName="%(RecursiveDir)%(Filename)%(Extension)" />
     </ItemGroup>
-    <ItemGroup>
-        <MauiXaml Update="Pages\BarcodeResultPage.xaml">
-            <SubType></SubType>
-        </MauiXaml>
-        <MauiXaml Update="Pages\HomePage.xaml">
-            <SubType></SubType>
-        </MauiXaml>
-        <MauiXaml Update="App.xaml">
-            <SubType></SubType>
-        </MauiXaml>
-    </ItemGroup>
     <ItemGroup>
         <Folder Include="Pages\" />
         <Folder Include="Models\" />
-        <Folder Include="ClassicComponent\" />
-        <Folder Include="Platforms\Android\Resources\layout\" />
-        <Folder Include="Platforms\iOS\Utils\" />
         <Folder Include="Common\" />
     </ItemGroup>
-    <ItemGroup Condition="$(TargetFramework.StartsWith('net7.0-android')) != true">
-        <Compile Remove="**\**\*.Android.cs" />
-        <None Include="**\**\*.Android.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
-    </ItemGroup>
-        <ItemGroup Condition="$(TargetFramework.StartsWith('net7.0-ios')) != true">
-        <Compile Remove="**\**\*.iOS.cs" />
-        <None Include="**\**\*.iOS.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
-    </ItemGroup>
     <!-- ===================== Android Project References =====================  -->
     <ItemGroup>
         <PackageReference Include="Scanbot.Xamarin.Forms.ImagePicker" Version="0.0.3-beta.1" />
-        <PackageReference Include="ScanbotBarcodeSDK.MAUI" Version="4.2.0" />
-    </ItemGroup>
-    <ItemGroup>
-      <AndroidResource Include="Platforms\Android\Resources\layout\barcode_camera_view.xml" />
+        <PackageReference Include="ScanbotBarcodeSDK.MAUI" Version="4.2.2-alpha.4" />

```

Comment:
@yurii-scanbot, we should add <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.3" /> as per warning MA002